views/index-getInvolved.html

```diff
@@ -1,5 +1,6 @@
-<div class="getInvolvedSection container">
+<div class="getInvolvedSection">
   <h2>Get Involved</h2>
+  <div class="container">
 
   <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```

Comment:
If you are using Sublime Text, you can highlight the lines, starting with the parent element, and then go to `Edit > Line > Reindent`.